modules/storage-azure-create-storage-class.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage-azure.adoc
+
+
+
+[id="storage-create-azure-storage-class_{context}"]
+= Creating the Azure storage class
+
+StorageClasses are used to differentiate and delineate storage levels and
```

Comment:
Is this correct as "StorageClasses"? Or should it just be "Storage classes" like elsewhere?

Reply:
Checking the glossary it looks like `StorageClasses` is correct.